libs/wingsdk/src/target-tf-azure/bucket.inflight.ts

```diff
@@ -7,15 +7,18 @@ export class BucketClient implements IBucketClient {
   private readonly bucketName: string;
   private readonly blobServiceClient: BlobServiceClient;
   private readonly containerClient: ContainerClient;
+  private readonly _public: boolean;
   private readonly defaultAzureCredential: DefaultAzureCredential =
     new DefaultAzureCredential();
 
   constructor(
     bucketName: string,
     storageAccount: string,
+    isPublic: boolean = false,
```

Comment:
I know! I prefer `isXXX` too :) 
The reason I did that in this place is this error: 
<img width="829" alt="image" src="https://user-images.githubusercontent.com/39455181/223865016-4ec3d595-b1cb-485a-b83b-702b18a300c2.png">
if you know a better way to prevent it can be awesome :)  (I'm won't think I'll go with _public argument in the constructor because `_XXX` is used most of the time for private properties as far as I know.... )

Reply:
I think isPublic is fine and if we need to get around the "isPublic" is declared but its value is never read then we can just add `isPublic;` to the first line of the method. (unless I misunderstood)